CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.17.0 Release
+- Release Date: March 16, 2019
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Combatibility
+ - If **DisableLegacyMfa** setting in ``config.json`` is set to ``true`` and [multi-factor authentication](https://docs.mattermost.com/deployment/auth.html) is enabled, ensure your users have upgraded to mobile app version 1.17 or later. See [Important Upgrade Notes](https://docs.mattermost.com/administration/important-upgrade-notes.html) for more details.
```

Comment:
Is the important upgrade notes doc updated? Don't see anything recent in there for this

Reply:
Yes... v5.9 docs haven't been merged yet as we're not ready to cut final yet

Reply:
And to verify, is this setting enabled or disabled by default in v5.9? I think disabled?

Reply:
Disabled, but it's still good to have the note if admins change it to True